snaps/reference/cli/subcommands.md

```diff
@@ -29,6 +29,25 @@ Builds a Snap from source.
 
 `b` is an alias for `build`.
 
+#### `analyze`
+
+<Tabs>
+<TabItem value="Syntax">
+
+```bash
+yarn mm-snap build --analyze
+```
+
+</TabItem>
+</Tabs>
+
+Enables analyzing the Snap bundle.
```

Comment:
Should we add information here? What gets produced when we use this? Why would we use this? 
Do we need to reference this anywhere in the Snaps content?